src/blob/core.rs

```diff
@@ -153,6 +153,22 @@ impl Blob {
             .context("failed to create iterator for raw records")
     }
 
+    async fn dump_corrupted(&self) -> Result<()> {
+        warn!("Blob {} dump started!", self.name());
+        let source_name = self.name.to_path();
+        let file_name = source_name
+            .file_name()
+            .ok_or_else(|| anyhow::anyhow!("Blob path is empty"))?;
+        let dir = self.name.dir.join("corrupted");
```

Comment:
Hardcoding this name is not good.  Declare public constant or get this name from Pearl config